qt/MainWindow.cc

```diff
@@ -1244,6 +1245,12 @@ void MainWindow::refreshPref(int key)
             break;
         }
 
+    case Prefs::READ_CLIPBOARD:
+        {
```

Comment:
braces not needed?  Variable `b` is already declared so we're not scoping anything here.

Mentioning it because I know Sonarcloud is going to hate on this :smiley:

Reply:
Removed braces